src/client/cypress/e2e/editor/bulkedit.cy.js

```diff
@@ -1,4 +1,44 @@
-import { createBorehole, login } from "../testHelpers";
+import { createBorehole, login, adminUserAuth } from "../testHelpers";
+
+export const changeWorkgroupRoleForUser = (
+  workgroupId,
+  userId,
+  role,
+  action,
```

Comment:
PP: Da die übergebenen Parameter bis auf `action` immer dieselben sind, würde ich diese inlinen und in aussagekräftige Variablen packen, z.B. `toggleWorkgroupRoleForEditUser = action => ...`